collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadListActivity.java

```diff
@@ -597,33 +572,55 @@ public void formListDownloadingComplete(HashMap<String, ServerFormDetails> resul
             toggleButtonLabel(toggleButton, listView);
 
             if (viewModel.isDownloadOnlyMode()) {
-                //1. First check if all form IDS could be found on the server - Register forms that could not be found
+                performDownloadModeDownload();
+            }
+        } else {
+            switch (exception.getType()) {
+                case FETCH_ERROR:
```

Comment:
We could also use the new less granular/more human errors for form download as well, but we'd need to discuss how that works. Collect is currently really verbose and outputs the failure/success status for each form so it's not as simple as just switching out the text.